src/components/functionselect/functionselect.js

```diff
@@ -62,24 +65,11 @@ angular.module('vlui')
             'variance', 'variancep'
           ] // hide COUNT for Q in the UI because we dedicate it to a special "# Count" field
         };
-        aggregates.all = aggregates.aboveFold.concat(aggregates.belowFold)
-          .concat([COUNT]); // COUNT is a valid aggregate
+        quantitativeFunctions.all = _.concat(quantitativeFunctions.aboveFold, quantitativeFunctions.belowFold);
```

Comment:
In fact, `quantitativeFunctions` never get used.  I don't know why you need to declare it at all.  Why don't merge it into the logic below.